hooks/useFetch.js

```diff
@@ -108,18 +104,27 @@ const useFetch = (url, refresh = false) => {
   const [loading, setLoading] = useState(true)
 
   useEffect(() => {
-    const { loading, data, refresh: startRefresh } = fetchData(url)
-    const sub = loading.subscribe(setLoading).add(
-      data.subscribe((ev) => {
-        setData(ev.value.body)
-        setError(ev.error)
+    const fetcher = fetchData(url).pipe(
```

Comment:
Will this toggle loading on refresh? I like the pattern here more, but loading will always just be `false` after the first time right?

Reply:
Yes. It won't toggle it, it will work like an argument to instantiate a consumable stream to either be one or the other. Did you want streams to allow toggling `refresh`? Can probably make that happen.

Reply:
Oh shit... i think I misunderstood and it just became clear to me. You're right I forgot to toggle `loading` when a refresh is attempted. Let me fix that.